bin/start

```diff
@@ -13,6 +13,12 @@ if [[ ! $SERVER_PORT ]]; then
   export SERVER_PORT=5000
 fi
 
+if [ -z "$DECENTRALAND_DATA" ]; then
```

Comment:
As the whole project is about Decentraland, I think `DATA_DIR` would make a better name, what do you think?